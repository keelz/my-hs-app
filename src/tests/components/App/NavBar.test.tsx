import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import AppNavBar from '../../../components/App/NavBar';

describe('AppNavBar', () => {
  describe('snapshots', () => {
    it('renders correctly', () => {
      const tree = enzyme.shallow(<AppNavBar />);
      expect(shallowToJson(tree)).toMatchSnapshot();
    });
  });
});
