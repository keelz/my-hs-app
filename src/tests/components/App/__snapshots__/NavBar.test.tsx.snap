// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavBar snapshots renders correctly 1`] = `
<Fragment>
  <Link
    replace={false}
    style={
      Object {
        "paddingRight": 10,
      }
    }
    to="/"
  >
    Home
  </Link>
  <Link
    replace={false}
    to="/test"
  >
    Test
  </Link>
</Fragment>
`;
